sdk/ai/azure-ai-projects/azure/ai/projects/models/_patch.py

```diff
@@ -726,6 +730,61 @@ def execute(self, tool_call: Any):
         """
 
 
+class AzureFunctionTool(Tool):
+    """
+    A tool that launches the Azure function.
```

Comment:
launches the Azure function can be little misleading, consider other wording

Reply:
Fixed